packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -18,7 +18,10 @@
 
 .position-container {
   @include floating-ui-container();
-  max-inline-size: var(--calcite-popover-max-size-x, 100vw);
+
+  & {
```

Comment:
@driskull I opted into the new behavior to address the warning ([produces separate selector](https://sass-lang.com/playground/#eJxtjsEOwiAQRO98xZ5M27i0XO2lv0IQzSZ0IS1Foum/C3rS9DbJvMw8paaZMjHcnNeR+I4bofEcNbFd4CUA+g6klND1Jc86I7ErHa70tBdQw5Ae4w+1CyGDXymS57+pidi47WqP35p2FAU6fdCDr6SXBtFoZyhaDD74ZBesWO0xn782bdXZi8YbQFBD+A==)), but we should look into updating some mixins to allow some CSS props overrides.